Package.swift

```diff
@@ -99,12 +99,13 @@ let package = Package(
     .package(
       name: "GoogleAppMeasurement",
       url: "https://github.com/google/GoogleAppMeasurement.git",
-      .exact("7.5.0")
+      .exact("7.5.1-test")
```

Comment:
Will it be removed before merge? Or it will have to be updated at the release time?

Reply:
Sorry, posted comment before reading the description :(

Reply:
It's a bit tricky since changing the names is technically a breaking change and the SPM install will be broken for everyone between the GoogleUtilities tag update and the firebase-ios-sdk tag update.  Going with the patch update since it fixes a 7.5.0 breakage on a beta distribution.